airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/bigquery/exchange_rate/models/generated/airbyte_tables/test_normalization/test_normalization_dedup_exchange_rate_scd.sql

```diff
@@ -9,11 +9,11 @@ select
     USD,
     date as _airbyte_start_at,
     lag(date) over (
-        partition by id, currency, date, cast({{ 'NZD' }} as {{ dbt_utils.type_string() }})
+        partition by id, currency, date, cast(NZD as {{ dbt_utils.type_string() }})
```

Comment:
`NZD` is a currency column and of type float. (we made it as a primary key it for the sake of the test)

In bigquery, you can't partition by float columns, so it is casted to string.

Unfortunately there was extra jinja_call & quotes resulting in a slightly different output...

Reply:
Actually the final SQL is exactly the same whether the dbt model is using:
- `cast({{ 'NZD' }} as {{ dbt_utils.type_string() }}`
- `cast(NZD as {{ dbt_utils.type_string() }}`

So there were no bugs here!!

(in this PR, we don't see the final SQL files being modified)

The user was in version 0.20.0 without this PR: #2919 so the model was actually `cast('NZD' as {{ dbt_utils.type_string() }}` which is translated to literal constant string...